src/main/vue/src/views/SoundView.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div>
+    <img v-if="imgFile" :src="require(`@/assets/images/${imgFile}`)" class="sound-img">
+  </div>
+</template>
+
+<script lang="ts">
+  import { Component, Vue } from 'vue-property-decorator';
+  import SoundsList from '@/components/category/SoundsList.vue';
+  import { Getter } from 'vuex-class';
+
+  @Component({
+    components: {
+      SoundsList,
+    },
+  })
+  export default class SoundView extends Vue {
+
+    private imgFile!: string;
```

Comment:
imgFile can be extracted from Sound, so I think there is no point in passing it as prop.